src/error-handling/converting-error-types.md

```diff
@@ -36,7 +49,7 @@ impl From<io::Error> for ReadUsernameError {
 
 fn read_username(path: &str) -> Result<String, ReadUsernameError> {
     let mut username = String::with_capacity(100);
-    fs::File::open(path)?.read_to_string(&mut username)?;
+    File::open(path)?.read_to_string(&mut username)?;
```

Comment:
Please don't change the existing code without a good reason. This change just leads to more imports at the top and it's harder to follow for the students who don't know the standard library very well.

Reply:
The existing code is inconsistent: some examples import `std::fs::File`, some import `fs::File` and use `fs::File` in the code. And other symbols are fully imported with their full paths, like `anyhow::Result` and `thiserror::Error`. The [documentation for `std::fs::File`](https://doc.rust-lang.org/std/fs/struct.File.html) also imports it with the full path.

Reply:
Yeah, I guess it has become a bit of a mess... My gut feeling is to use the module name when it's short, from the standard library. Which is not a very clear "rule" :smile: Based on that, I guess it's fine to import the names in the unqualified way you do here.